source/event_stream_rpc_client.c

```diff
@@ -500,7 +500,7 @@ static int s_send_protocol_message(
     struct event_stream_connection_send_message_args *args =
         aws_mem_calloc(connection->allocator, 1, sizeof(struct event_stream_connection_send_message_args));
 
-    if (!message_args) {
```

Comment:
If this wasn't found from specifically tracking down a bug, these are a hell of a catch!

Reply:
`aws_mem_calloc` can not fail. We should just get rid of this if block.